package.json

```diff
@@ -19,6 +19,7 @@
   "scripts": {
     "test": "start-server-and-test test:server http://localhost:8080 test:run",
     "test:server": "http-server _site -s",
-    "test:run": "swup validate"
+    "test:run": "swup validate",
+    "plugins:readme": "node ./lib/tasks/download-plugin-readmes.js"
```

Comment:
I would probably prefer "plugins:sync-readmes" or "sync:plugin-readmes" or the like. Not terribly important, though ;)

Reply:
Updated it, but kept the 'download'. I've wanted to keep the distinction between dumb downloading and potential future smart syncing.